### PR TITLE
Changed Offering schema and fixed styling

### PR DESCRIPTION
**Pull request by Cainuse**

### CODE REVIEW

File: models/Offering.js [hunk 1]
```diff
@@ -1,24 +1,41 @@
 const mongoose = require("mongoose");
 const Schema = mongoose.Schema;
-const OfferingSchema = Schema({
-  comment: {
+
+const Item = Schema({
```
Comment (jacquelineyin) [inline models/Offering.js:4]: I'm fine with this, but just to check:

on FE, everything other than "images" is required for items. Should I remove that validation check and let it be optional? Or should I just leave it as it is? (I don't think BE needs to make them required)

**Review by jacquelineyin — APPROVED** [review]
Looks good to me! Just want to check on the question that I tagged in the code somewhere, but aside from confirmation on that, I think it's good for a merge :D

File: models/Offering.js (hunk 1 above)
Comment (abid-salahi) [reply models/Offering.js:4]: I think its fine to leave comments optional as not everyone might actually write something.

Comment (jacquelineyin) [reply models/Offering.js:4]: It's true, but if the user doesn't write anything in the comments, FE still always returns `comment: ""`, so I think it's fine if Peng wants to keep it as required. (Either option works, I think)

**Pull request merged**